roles/tomcat7/tasks/tomcat_user_instance.yml

```diff
@@ -0,0 +1,88 @@
+---
+- name: Create Tomctat System User
+  become: yes
+  become_user: root
+  user:
+    name: "{{ tomcat_system_user }}"
+    group: "{{ tomcat_system_group }}"
+    home: "{{ tomcat_user_home }}"
+    shell: /bin/bash
+    state: present
+  tags:
+   - tomcat-user-instance
+
+- name: Create Tomcat7 User Instance
+  become: yes
+  become_user: "root"
+  command: "tomcat7-instance-create {{ tomcat_instance }}"
+  args:
+    chdir: "{{ tomcat_user_home }}"
+    creates: "{{ tomcat_user_home }}/{{ tomcat_instance }}"
+  tags:
+   - tomcat-user-instance
+
+- name: Copy Tomcat7 setenv Script
+  become: yes
+  become_user: "root"
+  template:
+    src: "templates/tomcat_user_instance_home/bin/setenv.sh.j2"
+    dest: "{{ tomcat_user_home }}/{{ tomcat_instance }}/bin/setenv.sh"
+    owner: "{{ tomcat_system_user }}"
+    group: "{{ tomcat_system_group }}"
+    mode: 0744
+  notify:
+    - "restart {{ tomcat_instance }}"
+  tags:
+   - tomcat-user-instance
+
+- name: Copy Tomcat7 Configs
```

Comment:
If this task ever changed the files, you'd probably want to notify the service restart, no?